src/components/Tabs.tsx

```diff
@@ -29,33 +29,40 @@ export interface TabsProps<V extends string, X extends Properties> {
  *
  * The caller is responsible for using `selected` / `onChange` to control
  * the current tab.
+ *
+ * If you want to tease apart Tabs from their TabContent, you can use the `Tab`
+ * and `TabContent` components directly.
  */
 export function TabsWithContent<V extends string, X extends Only<TabsContentXss, X>>(props: TabsProps<V, X>) {
-  const { selected, tabs, contentXss = {}, ...others } = props;
-  const selectedTab = tabs.find((tab) => tab.value === selected) || tabs[0];
-  const tid = useTestIds(others, "tab");
-
-  const onlyOneTabEnabled = tabs.filter((t) => !t.disabled).length === 1;
-
+  const onlyOneTabEnabled = props.tabs.filter((t) => !t.disabled).length === 1;
   return (
     <div>
```

Comment:
**suggestion:** Can we remove this surrounding div and replace with a fragment while we're in here?

Reply:
Yep, done